src/library/SetupSteps/types.ts

```diff
@@ -21,7 +21,7 @@ export interface GenerateNominationsInnerProps {
 }
 
 export interface HeaderProps {
-  title?: string;
+  title?: string | null;
```

Comment:
Why  `null` added?

Reply:
Same type error we just spoke of

Reply:
the error was with `string | undefined` wasn't it? Which we ensured wouldn't happen with ` || ''`.

Does the code work if you remove null from this type?